### PR TITLE
Fix the translation omissions in the scientific objects creation menu

### PR DESCRIPTION
**Pull request by pierreetiennealary**

### CODE REVIEW

File: translations/fr/messages.php [hunk 1]
```diff
@@ -46,6 +46,7 @@
     'The error above occurred while the Web server was processing your request.' => 'L\'erreur ci-dessus est survenue lors du traitement de votre requête par le serveur web.',
     'The value' => 'La valeur',
     'The variety used in the plot (e.g. apache)' => 'La variété du plot (ex. apache)',
+	'to get more informations about the columns contents' => 'pour obtenir davantage d\'information sur le contenu de chaque colonne',
```
Comment (Andreas-Garcia) [inline translations/fr/messages.php]:
```suggestion
	'to get more informations about the columns content' => 'pour obtenir davantage d\'information sur le contenu de chaque colonne',
```

File: views/scientific-object/_form.php [hunk 2]
```diff
@@ -18,7 +18,12 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/handsontable/0.37.0/handsontable.full.min.js"></script>
 
 <div class="scientificObject-form">
-    <p><i>See the <a href="https://opensilex.github.io/phis-docs-community/experimental-organization/#importing-scientific-objects" target="_blank">documentation</a> to get more informations about the columns contents.</i></p>
+    <p><i>
+	<?= Yii::t('app', 'See') ?>
+	<a href="https://opensilex.github.io/phis-docs-community/experimental-organization/#importing-scientific-objects" target="_blank"><?= Yii::t('app', 'the documentation') ?></a>
+	<?= Yii::t('messages', 'to get more informations about the columns contents') ?>
```
Comment (Andreas-Garcia) [inline views/scientific-object/_form.php]:
```suggestion
	<?= Yii::t('messages', 'to get more information about the columns content') ?>
```

File: translations/fr/messages.php [hunk 3]
```diff
@@ -46,6 +46,7 @@
     'The error above occurred while the Web server was processing your request.' => 'L\'erreur ci-dessus est survenue lors du traitement de votre requête par le serveur web.',
     'The value' => 'La valeur',
     'The variety used in the plot (e.g. apache)' => 'La variété du plot (ex. apache)',
+	'to get more informations about the columns content' => 'pour obtenir davantage d\'information sur le contenu de chaque colonne',
```
Comment (Andreas-Garcia) [inline translations/fr/messages.php]:
```suggestion
	'to get more information about the columns content' => 'pour obtenir davantage d\'information sur le contenu de chaque colonne',
```

File: translations/fr/app.php [hunk 4]
```diff
@@ -98,8 +98,10 @@
     'Financial Support' => 'Support financier',
     'Focal Length' => 'Distance Focale',
     'Height' => 'Hauteur',
+	'Insertion status' => 'Statut de l\'insertion',
```
Comment (annetireau) [inline translations/fr/app.php]:
```suggestion
'Insertion status' => 'Statut de l\'insertion',
```

File: translations/fr/app.php [hunk 5]
```diff
@@ -98,8 +98,10 @@
...
     'Generate Layer' => 'Générer la Couche',
     'Generate Map' => 'Générer la Carte',
+	'Generated URI' => 'URI généré',
```
Comment (annetireau) [inline translations/fr/app.php]:
```suggestion
'Generated URI' => 'URI généré',
```

File: translations/fr/app.php [hunk 6]
```diff
@@ -151,6 +153,7 @@
     'Replication' => 'Répétition',
     'Scientific Contacts' => 'Contacts scientifiques',
     'Scientific Supervisors' => 'Superviseurs scientifiques',
+	'See' => 'Voir',
```
Comment (annetireau) [inline translations/fr/app.php]:
```suggestion
'See' => 'Voir',
```

File: translations/fr/app.php [hunk 7]
```diff
@@ -163,11 +166,13 @@
     'Show Images' => 'Afficher les Images',
     'Spectral hemispheric reflectance file' => 'Fichier de réflectance hémisphérique spectrale',
     'Spectralon' => 'Spectralon',
+	'Species' => 'Espèce',
```
Comment (annetireau) [inline translations/fr/app.php]:
```suggestion
'Species' => 'Espèce',
```

File: translations/fr/app.php [hunk 8]
```diff
@@ -163,11 +166,13 @@
...
     'Status' => 'Statut',
     'Subproject Type' => 'Type de sous projet', 
     'Subproject Of' => 'Sous-projet de',
     'Team' => 'Équipe',
     'Technical Supervisors' => 'Superviseurs techniques',
+	'the documentation' => 'la documentation',
```
Comment (annetireau) [inline translations/fr/app.php]:
```suggestion
'the documentation' => 'la documentation',
```

File: translations/fr/messages.php (hunk 3 above)
Comment (annetireau) [inline translations/fr/messages.php]:
```suggestion
'to get more informations about the columns content' => 'pour obtenir davantage d\'information sur le contenu de chaque colonne',
```

Comment (pierreetiennealary) [reply translations/fr/messages.php]: this line has already been changed by Andreas

**Comment by annetireau** [comment]
Please, could you check the indentation

**Pull request closed without merging**